### PR TITLE
Point layer norm to torch documentation

### PR DESCRIPTION
**Pull request by joecummings**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [ ] fix a bug
- [x] update tests and/or documentation
- [ ] other (please add here)

Please link to any issues this PR addresses.

#### Changelog

* Point function reference to torch documentation

#### Test plan

EYES

### CODE REVIEW

File: torchtune/modules/layer_norm.py
Comment (RdoubleA) [inline torchtune/modules/layer_norm.py:15]: i think you still need to keep torch in the path here?

**Review by RdoubleA — APPROVED** [review]

**Comment by joecummings** [comment]
@RdoubleA Don't merge this PR - leaving it for @SalmanMohammadi to test his new merge rights.

**Comment by SalmanMohammadi** [comment]
Thanks for padding my GitHub stats!

**Pull request merged**
